guide/daily-spending-wallet/sending.md

```diff
@@ -165,17 +165,17 @@ When responding to an invoice that contains all relevent information, the user c
 
 **Recipient**
 
-The most convenient option for choosing a recipient is from previously saved contacts. Alternatively, users can enter on-chain addresses, Lightning addresses, Lightning node IDs, or other static identifies that are supported by the wallet.
+The most convenient option for choosing a recipient is from a previously saved [contact]({{ '/guide/daily-spending-wallet/contacts/' | relative_url }}). Alternatively, users can enter Lightning addresses, Lightning node IDs, on-chain addresses, or other addresses that are supported by the wallet.
 
-There are also static [invoice types]({{ '/guide/how-it-works/payment-request-formats/' | relative_url }}) that can receive payments repeatedly. These are less intuitive overall due to their appearance, but could also be considered payment endpoints.
+There are also static [payment requests]({{ '/guide/how-it-works/payment-request-formats/' | relative_url }}) that can receive payments repeatedly. These are less intuitive overall due to their appearance, but could also be considered payment endpoints.
 
 **Amount**
 
 If no amount is provided via a payment request, manual entry should be simple and convenient so users don’t accidentally send an incorrect amount. The amount should be displayed in bitcoin or satoshi value, as well as the user’s local currency. Options to quickly toggle between them should be available. More on the [Units & Symbols page]({{ '/guide/designing-products/units-and-symbols/' | relative_url }}).
 
 **Metadata**
 
-A transaction history is hard to make sense of when it only shows amounts, dates, and identifiers. Users should be allowed to add descriptions, tags, and other metadata to add context. This context can separately be used for helpful tools like visual spending breakdowns.
+A [transaction history]({{ '/guide/daily-spending-wallet/activity/' | relative_url }}) is hard to make sense of when it only shows amounts, dates, and identifiers. Users should be allowed to add descriptions, tags, and other metadata to add context. This context can separately be used for helpful tools like visual spending breakdowns.
 
 ## Fees
```

Comment:
"Payment fees can drastically differ based on a few attributes:" is getting covered up by the black fees picture on my monitor. Sentence needs to be highlighted to read it.

![Screen Shot 2022-05-24 at 4 18 48 PM](https://user-images.githubusercontent.com/2414177/170125004-8076e7bf-7071-42d8-83c6-269d111803f4.png)

![Screen Shot 2022-05-24 at 4 20 42 PM](https://user-images.githubusercontent.com/2414177/170125003-33a0fbb8-0e5d-4563-8754-a8ff8a0fd5ad.png)

Reply:
Good catch. This has to do with our deep-link-into-a-header-spacing-to-ensure-that-the-title-is-not-covered-up-by-the-static-nav-bar fix. I reverted the change that switched the bold titles to header tags.